sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java

```diff
@@ -377,7 +285,7 @@ public Long countBySearch(String querystr) throws IOException {
     query.addOrder("asc".equalsIgnoreCase(sortDir) ? Order.asc(sortCol) : Order.desc(sortCol));
```

Comment:
It would be nice if all instances of `query` could be renamed to `criteria` (and converted to Criteria if they're not, but it looks like you've taken care of those)